scenes/game_elements/characters/player/components/player.gd

```diff
@@ -37,6 +37,8 @@ const DEFAULT_SPRITE_FRAME: SpriteFrames = preload(
 @export var sprite_frames: SpriteFrames = DEFAULT_SPRITE_FRAME:
 	set = _set_sprite_frames
 
+var input_vector: Vector2
```

Comment:
This variable name could have the private variable convention `_input_vector` (prepend underscore).

Reply:
I didn't do this because I imagined that, in future, this could be set from `PlayerController` (so that that node can be paused independently of the rest of the player) and because now it would be possible to animate the player's movement in cutscenes by adjusting `input_vector`.